src/test/python/apache/aurora/config/test_loader.py

```diff
@@ -61,6 +64,11 @@
 jobs = [HELLO_WORLD, OTHERJOB]
 """
 
+MESOS_CONFIG_WITH_INCLUDE_TEMPLATE = JOB2 + """
```

Comment:
nit. Should the file be just
```
include("./%s")
jobs = [HELLO_WORLD, OTHERJOB]
```
I think `JOB2 + ` is not needed?

Reply:
nvm... I think the intention here was to have a mix of included config and normal config.

Reply:
It is needed JOB2 defines OTHERJOB. The test then includes another file defining the HELLO_WORLD job